### PR TITLE
Add fetch argument to build command.

### PR DESCRIPTION
**Pull request by dabrain34**
it allows to check all recipes source state and invalidate the build
if necessary on each of those where the source state has changed.

### CODE REVIEW

**Comment by dabrain34** [comment]
To replace https://github.com/fluendo/cerbero/pull/176

File: cerbero/commands/build.py [hunk 1]
```diff
@@ -92,9 +94,14 @@ def runargs(self, config, recipes, missing_files=False, force=False,
             ordered_recipes = recipes
         else:
             ordered_recipes = cookbook.list_recipes_deps(recipes)
+        if fetch:
+          self._fetch(config, ordered_recipes, no_deps=False, reset_rdeps=True)
         oven.start_cooking(ordered_recipes, use_binaries, upload_binaries,
                 build_missing)
 
+    def _fetch(self, config, recipes, no_deps=False, reset_rdeps=False):
```
Comment (ylatuya) [inline cerbero/commands/build.py]: no_deps and reset_rdeps are not needed as it's always used with the same values.

Comment (dabrain34) [reply cerbero/commands/build.py]: Yes I prefered it moludar

Comment (dabrain34) [reply cerbero/commands/build.py]: > We should try to upstream this first

Ok

Comment (ylatuya) [reply cerbero/commands/build.py]: yes, we are already upstreaming things, I don't understand the question :) GitLab's account and fluendo's email in the commits.

**Pull request closed without merging**